common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -447,6 +452,15 @@ private boolean isAmazonAppRedirect(@NonNull final String url) {
         return url.startsWith(AMAZON_APP_REDIRECT_PREFIX);
     }
 
+    private boolean isWebCpEnrollmentUrl(@NonNull final String url) {
+        return url.startsWith(AuthenticationConstants.Broker.WEBCP_ENROLLMENT_URL);
+    }
+
+    private boolean isWebCpAuthorizeUrl(@NonNull final String url) {
+        // URL is for an authorize request and contains the client_id of the WebCP app.
+        return url.contains(AuthenticationConstants.Broker.WEBCP_AUTHORIZE_URL_PATTERN);
```

Comment:
should we also check domain name - especially since we're attaching PRT header in the request?

(e.g. www.someweb.com/authorize?client_id=74bcdadc-2fdc-4bb3-8459-76d06952a0e9 would return true)

Reply:
Perhaps could use a similar check with isCrossCloudRedirect()